sign/schnorr/schnorr.go

```diff
@@ -60,6 +65,9 @@ func Sign(s Suite, private kyber.Scalar, msg []byte) ([]byte, error) {
 // Verify verifies a given Schnorr signature. It returns nil iff the
 // given signature is valid.
 func Verify(g kyber.Group, public kyber.Point, msg, sig []byte) error {
+	if public.Equal(g.Point().Null()) {
+		return errors.New("invalid public key")
```

Comment:
Giving an error if you want to sign with a zero key makes sense. Just asking - does it make sense to forbid checking the signature with a Null point? It's always true...